### PR TITLE
clean makefile

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,11 @@
-.PHONY: check clean install source help
+.PHONY: test ci-test clean install source
 
-check:
+test:
```
Comment (praiskup) [inline Makefile]: :) make users love `check`

Comment (TomasTomecek) [reply Makefile]: we agreed today that all our makefiles will have `test` target to run tests; we can easily add `check: test`, I wouldn't mind

Comment (praiskup) [reply Makefile]: I don't mind either; I'm just saying that `check` is far more standard.

Comment (TomasTomecek) [reply Makefile]: thanks for heads up then! I'll open an issue for it

**Pull request merged**